UnitTests/BraintreeTestShared/FakeHTTP.swift

```diff
@@ -35,102 +35,104 @@ import BraintreeCore.Private
         cannedError = error
     }
 
-    public override func get(_ endpoint: String, parameters: [String : String]?, completion completionBlock: ((BTJSON?, HTTPURLResponse?, Error?) -> Void)? = nil) {
+    public override func get(_ path: String, parameters: NSDictionary? = nil, completion: ((BTJSON?, HTTPURLResponse?, Error?) -> Void)? = nil) {
         GETRequestCount += 1
-        lastRequestEndpoint = endpoint
+        lastRequestEndpoint = path
         lastRequestParameters = parameters
         lastRequestMethod = "GET"
 
         if cannedError != nil {
             dispatchQueue.async {
-                completionBlock!(nil, nil, self.cannedError)
+                completion!(nil, nil, self.cannedError)
```

Comment:
Good catch, we do a lot of force unwrapping in our tests in general that need to be addressed at some point. 🙃 For these, since we are defaulting them to `nil` I think we can make them all optional completions vs force unwrapping them.